public/app/features/alerting/components/AlertingQueryPreview.tsx

```diff
@@ -1,26 +1,29 @@
 import React, { FC, useMemo, useState } from 'react';
 import { useObservable } from 'react-use';
 import { css } from 'emotion';
+import AutoSizer from 'react-virtualized-auto-sizer';
 import { GrafanaTheme } from '@grafana/data';
-import { TabsBar, TabContent, Tab, useStyles, Table } from '@grafana/ui';
+import { TabsBar, TabContent, Tab, useStyles, Icon } from '@grafana/ui';
 import { PanelQueryRunner } from '../../query/state/PanelQueryRunner';
+import { PreviewQueryTab } from './PreviewQueryTab';
+import { PreviewInstancesTab } from './PreviewInstancesTab';
 
 enum Tabs {
   Query = 'query',
   Instance = 'instance',
 }
 
 const tabs = [
-  { id: Tabs.Query, text: 'Query', active: true },
-  { id: Tabs.Instance, text: 'Alerting instance', active: false },
+  { id: Tabs.Query, text: 'Query' },
+  { id: Tabs.Instance, text: 'Alerting instance' },
 ];
 
 interface Props {
   queryRunner: PanelQueryRunner;
 }
 
 export const AlertingQueryPreview: FC<Props> = ({ queryRunner }) => {
-  const [activeTab, setActiveTab] = useState<string>('query');
+  const [activeTab, setActiveTab] = useState<string>(Tabs.Query);
   const styles = useStyles(getStyles);
 
   const observable = useMemo(() => queryRunner.getData({ withFieldConfig: true, withTransforms: true }), []);
```

Comment:
After a brief discussion with Peter I realized that we need `useMemo` because how `useObservable` works, it has a hook dependency on the observable so that needs to be memoized in this case (maybe in all cases, not sure it works as is). 

Writing this without `useObservable` means a bit more code but is clearer what is going on and which hook dependencies are used:
```ts
const [data, setData] = useState<PanelData>({
    series: [],
    state: LoadingState.NotStarted,
    timeRange: getDefaultTimeRange(),
  });

  useEffect(() => {
    const subscription = queryRunner
      .getData({ withFieldConfig: true, withTransforms: true })
      .subscribe(value => setData(value));

    return function unsubscribe() {
      return subscription.unsubscribe();
    };
  }, [queryRunner]);
```